src/kabr_tools/cvat2ultralytics.py

```diff
@@ -193,12 +195,25 @@ def parse_args():
         help='process one out of skip number of frames',
         default=10
     )
+    local_parser.add_argument(
+        '--label2index',
+        type=str,
+        help='path to label to index json',
```

Comment:
```suggestion
        help='path to label to index json (default is for zebra, baboon, and giraffe)',
```
Some indication of the default can be helpful.